terraform/stacks/main/stack.tf

```diff
@@ -124,6 +124,26 @@ module "cf" {
     )}"]
 }
 
+module "credhub" {
+    source = "../../module/credhub"
+    
+    stack_description = "${var.stack_description}"
+
+    elb_main_cert_id = "${data.aws_iam_server_certificate.wildcard.arn}"
```

Comment:
Looks like this variable isn't used in the `credhub` module. Can you drop any unused variables here?